.github/scripts/review_agent.py

```
@@ -1,65 +1,27 @@
-    ("nvidia/nemotron-3-super-120b-a12b", "https://integrate.api.nvidia.com/v1", os.environ.get("NVIDIA_API_KEY")),
-    ("kimi-k2.6", "https://api.moonshot.cn/v1", os.environ.get("MOONSHOT_API_KEY")),
-]
-
 def main():
-    """
-    Run a PR review using available candidate LLM backends and write the verdict to RESULT_FILE.
-    
-    Fetches the PR diff for PR_NUMBER and prompts the models to review security and correctness using the expected `OVERALL: PASS|FAIL` format. Queries configured candidate models until one returns a response, writes JSON to RESULT_FILE containing `{"verdict": <"PASS"|"FAIL">, "summary": <first 200 chars of response>}`, and exits with status 0 when the verdict is PASS or 1 otherwise. If no model produces a response, exits with status 1.
-    """
+    client = OpenAI(base_url="https://integrate.api.nvidia.com/v1", api_key=os.environ["NVIDIA_API_KEY"])
     diff = subprocess.check_output(["gh", "pr", "diff", PR_NUMBER, "--patch"], text=True)[:10000]
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)